protocol/app/ve/aggregator/ve_aggregator.go

```diff
@@ -148,17 +150,43 @@ func GetDaemonVotesFromBlock(
 	veCodec codec.VoteExtensionCodec,
 	extCommitCodec codec.ExtendedCommitCodec,
 ) ([]Vote, error) {
-	if len(proposal) < constants.InjectedNonTxCount {
```

Comment:
why is this check removed?

Reply:
i think you made this change, if i did was a mistake/accidental